### PR TITLE
Small fixups

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: src/util/cmdline.rs [hunk 1]
```diff
@@ -62,7 +62,7 @@ fn contains_flag_prefix(cmdline: &str, prefix: &str) -> bool {
     cmdline.split(' ').any(|s| s.starts_with(prefix))
 }
 
-// Find OEM ID flag value in cmdline string.
+// Find argument value in cmdline string.
```
Comment (prestist) [inline src/util/cmdline.rs]: sorry just a nit, argument's ?

Comment (bgilbert) [reply src/util/cmdline.rs]: The function argument is `flagname` so I guess I should use `flag`.

**Comment by bgilbert** [comment]
Updated.

**Review by prestist — APPROVED** [review]
LGTM

**Pull request merged**